src/content/updates/v10-migration/designers.mdx

```diff
@@ -167,23 +146,30 @@ Layout and spacing token names have also been updated in v10 but the values rema
-Set up a meeting with your dev team and OM to discuss the migration process. Using this page as a starting point, establish a plan of attack for the migration together. 
+The Carbon design kit now lives in a Sketch library, which can be updated on a regular basis by the Carbon team. If you're working with a local document, visit the [Getting started](/getting-started/designers) guide and get set up with the new libraries. 
 
-Ideally, your product team will upgrade on a parallel branch, where any interface issues can be worked out between design and dev, before going GA with the update.
+The components and elements in the kit have been completely revised from v9. Be sure to note the changes to the color palette, spacing and layout, and iconography in the kit.
 
-### "Code-first" approach
+#### "Code-first" approach
 
 The fastest approach to v10 migration starts with the front-end developer. The developer, with help from the migration guidelines found on this site, will update the product's front end code to v10. This should not cause any major breaks in the UI, but there might be some funky looking stuff that needs to be resolved by the design team. Product teams using v9 components in their code should see a mostly seamless transition.  
 
-The product design team should review the updated UI with a critical eye and a solid knowledge of the v10 design guidelines. Do the fonts look right? Are the colors correct? Is the spacing between components correct? Do animations in the UI match the new motion standards? Does the general layout still work, and does it express the IBM Design Language accurately and effectively? Working together, the design and dev teams will fix any anomalies or glitches in the layout.
+The product design team should review the updated UI with a critical eye and a solid knowledge of the v10 design guidelines. Throughout the process, ask yourself: 
+- Do the fonts look right? 
+- Are the colors correct? 
+- Is the spacing between components correct? 
+- Do animations in the UI match the new motion standards? 
+- Does the general layout still work, and does it express the IBM Design Language accurately and effectively? 
+
+Working together, design and dev teams will catch and fix any problems with page layout and design.
 
 ## Feedback
 
 To leave feedback for the Carbon team, make suggestions for improvements, or just ask a question, you can [open an issue](https://github.com/carbon-design-system/carbon-website/issues/new?template=feedback.md&labels=feedback&title=User+feedback+for+Migration+Designers+page) on GitHub.
 
-Internal IBM users can also subscribe to the [#carbon-design-system Slack channel](https://ibm-studios.slack.com/messages/C0M053VPT/details/), which is monitored by the Carbon team and the broader Carbon community.
+IBMers should join the [#carbon-design-system Slack channel](https://ibm-studios.slack.com/messages/C0M053VPT/details/) to connect with the Carbon community, ask questions, and stay in the loop.
```

Comment:
Double checking that "IBMers" is aligned with IBM's content style guide for describing IBM employees.

Reply:
Resolved this!